src/app/services/blog.service.ts

```diff
@@ -22,7 +27,14 @@ export class BlogService {
           actions.map((a) => {
```

Comment:
What is a?  seems like data is already have createdAt, so this looks like a duplicate